src/api/v4/filter/bounding-box-gmaps.js

```diff
@@ -53,4 +59,9 @@ BoundingBoxGoogleMaps.prototype.$getInternalModel = function () {
   return this._internalModel;
 };
 
+// Helper to check if an element is a leafletmap object
+function _isGoogleMap (element) {
+  return element instanceof google.maps.Map;
```

Comment:
We're relying on `google` to be present but this check will throw an error if object `google` is not present.

In this case we should trigger an error asking the user to get google properly added to the document.